src/db_bench/main.rs

```diff
@@ -53,6 +56,18 @@ fn load_object_store(args: &DbBenchArgs) -> Result<Arc<dyn ObjectStore>, SlateDB
 
 #[tokio::main]
 async fn main() {
+
+    // Initialize the tracing subscriber
+    let subscriber = FmtSubscriber::builder()
+        .with_max_level(Level::TRACE)
```

Comment:
For benching, I think we probably want info-level default. With that in mind, perhaps this is easier:

```rust
tracing_subscriber::fmt::init();
```